internal/cri/server/container_create.go

```diff
@@ -1008,51 +1008,59 @@ func (c *criService) linuxContainerMounts(sandboxID string, config *runtime.Cont
 	}
 
 	if !isInCRIMounts(etcHosts, config.GetMounts()) {
-		mounts = append(mounts, &runtime.Mount{
-			ContainerPath:  etcHosts,
-			HostPath:       c.getSandboxHosts(sandboxID),
-			Readonly:       securityContext.GetReadonlyRootfs(),
-			SelinuxRelabel: true,
-			UidMappings:    uidMappings,
-			GidMappings:    gidMappings,
-		})
+		hostpath := c.getSandboxHosts(sandboxID)
+		if _, err := c.os.Stat(hostpath); err == nil {
```

Comment:
Would you please add comment to explain why we ignore not-found error here ? Thanks

Reply:
Should I add to all the places or just one? Actually, the comments are all same.

Reply:
Added one, PTAL.

Reply:
please add the comments to the following change. Thanks!

Reply:
Done

Reply:
Suggest to have

```
/etc/hosts could be delegated to remote sandbox controller. That file isn't required to be existed in host side for some sandbox runtimes. Skip it if we don't need it.
```